images_config/ks-robot-master.cfg

```diff
@@ -271,11 +271,15 @@ chmod +x /usr/local/bin/socket_buffer.py
 #initctl start candump_any
 %end
 
-################ INSTALL NOMACHINE ################ 
+################ INSTALL PULSEAUDIO ################ 
 %post --erroronfail --interpreter /bin/bash
 apt-get install pulseaudio -y
```

Comment:
should packages to be installed not be listed in [packages section](https://github.com/ipa-fmw/setup_cob4/blob/4e2bc5487e07f189d567cbbeb70e1958d9b126a7/images_config/ks-robot-master.cfg#L45) further up?

Reply:
done in https://github.com/ipa320/setup_cob4/pull/192/commits/34ad7962cec0444bb1500e24a25f335f95e3d251